arcgis-java-toolkit-examples/src/main/java/com/esri/arcgisruntime/toolkit/examples/FloorFilterExample.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2022 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.examples;
+
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.view.GeoView;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.toolkit.FloorFilter;
+import com.esri.arcgisruntime.toolkit.model.Example;
+import com.esri.arcgisruntime.toolkit.utils.ExampleUtils;
+import javafx.application.Application;
+import javafx.scene.Node;
+import javafx.scene.control.*;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An Example Class for the {@link FloorFilter} Toolkit Component. Implements the {@link Example} interface to ensure
+ * required methods are implemented. The example can be viewed by running the
+ * {@link com.esri.arcgisruntime.toolkit.ExamplesApp}, or as a standalone app via the {@link FloorFilterExampleLauncher}
+ * class.
+ *
+ * @since 100.15.0
+ */
+public class FloorFilterExample extends Application implements Example {
+
+    // a MapView and a FloorFilter to be used on the MapView
+    private final MapView mapView = new MapView();
+    private final FloorFilter floorFilter;
+
+    // the required UI components
+    private final List<Tab> tabs = new ArrayList<>();
+    private final VBox settings;
+    private final BorderPane borderPane;
+
+    /**
+     * A constructor for the FloorFilterExample that demonstrates how to implement a FloorFilter for a MapView.
+     *
+     * A Tab is created for the view to display in the application and settings are configured for testing and demo
+     * purposes.
+     *
+     * @since 100.15.0
+     */
+    public FloorFilterExample() {
+        // configure mapview tab
+        ArcGISMap map = new ArcGISMap("https://www.arcgis.com/home/item.html?id=f133a698536f44c8884ad81f80b6cfc7");
```

Comment:
some extra info for this comment would be good, e.g. some brief info on what the webmap contains